common/changes/@uifabric/experiments/pagination1001_2018-10-01-20-51.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/experiments",
+      "comment": "ainitial checkin to add pagination controldd pagination control",
```

Comment:
Nit but worth fixing since its the changelog message we publish. I'd change this to:

>Initial checkin to add pagination control

Reply:
addressed.

---
In reply to: [222067969](https://github.com/OfficeDev/office-ui-fabric-react/pull/6532#discussion_r222067969) [](ancestors = 222067969)